src/devtools/client/debugger/src/actions/breakpoints/index.js

```diff
@@ -208,7 +215,13 @@ export function addBreakpointAtColumn(cx, location) {
     };
 
     const options = {};
-    const file = source.url.split("/").pop();
+    let file = source.url?.split("/").pop();
+
+    if (!file) {
+      const { host } = getSourceNames(source.url || source.relativeUrl);
+      file = host;
+    }
```

Comment:
A couple of thoughts....

1. if there isn't a url, it's okay for the file to be empty ""
2. if we know there isn't a url, let's not try to get the source names. I doubt there's anything there.

Reply:
the reason i did that was because in the console, for a certain file's breakpoints, they would show up as ":15" for a breakpoint instead of "todomvc.com:15"

Reply:
hmm, will look on monday. perhaps this is for HTML index files

Reply:
yeah, it was for index. sounds good, chat then!